packages/cli/src/scripts/send-tx.ts

```diff
@@ -0,0 +1,11 @@
+import TransactionController from '../models/network/TransactionController';
+import { SendTxParams } from './interfaces';
+
+export default async function sendTx({ proxyAddress, methodName, methodArgs, value, gas, network, txParams, networkFile }: SendTxParams): Promise<void | never> {
```

Comment:
Why proxyAddress not checked? What if it is undefined?

Reply:
Good catch :+1: